funnel/views/helpers.py

```diff
@@ -192,6 +193,19 @@ def url_join(base, url=''):
     return urljoin(base, url)
 
 
+@app.template_filter('cleanurl')
+@funnelapp.template_filter('cleanurl')
+def cleanurl_filter(url):
+    url = url if isinstance(url, furl) else furl(url)
```

Comment:
Why not the idiomatic:
```python
if not isinstance(url, furl):
    url = furl(url)
```
Doing this will give a more accurate coverage report for the second line.